src/Core/SharedFolderProvider.cs

```diff
@@ -30,7 +30,7 @@ public interface ISharedFolderProvider
         /// Returns the file path to the current Shared Folder
         /// Or null if there's not a Shared Folder
         /// </summary>
-        string GetSharedFolderPathOrNull();
+        string GetSharedFolderPath();
```

Comment:
Added this change to this PR because it was already a too small PR and it didn't make sense to create another PR just for this..